requirements/pytorch/check-avail-strategies.py

```diff
@@ -1,7 +1,17 @@
-import bagua  # noqa: F401
-import deepspeed  # noqa: F401
-import fairscale  # noqa: F401
-import horovod.torch
+import sys
 
-# returns an error code
-assert horovod.torch.nccl_built()
+if __name__ == "__main__":
+    args = sys.argv[1:]
```

Comment:
Where are we making the modification to pass arguments to this? Maybe I'm blind but not seeing it

Reply:
it is in building docker images and later on GPU workflows

Reply:
well I do not see it either so would need to check what happened

Reply:
found it here: https://github.com/Lightning-AI/lightning/pull/15403/files#r1018509556